### DISCUSSION TITLE
Connector for Login with Amazon?

### DISCUSSION THREAD

**Discussion by jsshapiro** [dexidp/dex#1994] [Ideas]
Has anyone attempted to provision a connector to Login With Amazon? They are using an OAuth2 process that does much (not quite all) of the OIDC spec. We're going to need it, but I wanted to check here before digging in to do a connector.

**Comment by sagikazarmark** [comment]
Here is a PR that's due to be merged in the next release: https://github.com/dexidp/dex/pull/1630
